homeassistant/components/nest/api.py

```diff
@@ -58,7 +58,7 @@ async def async_get_creds(self) -> Credentials:
         # even when it is expired to fully hand off this responsibility and
         # know it is working at startup (then if not, fail loudly).
         token = self._oauth_session.token
-        creds = Credentials(  # type: ignore[no-untyped-call]
+        creds = Credentials(
```

Comment:
Looks like MyPy disagrees with this change
```suggestion
        creds = Credentials(  # type: ignore[no-untyped-call]
```